core-ui/src/components/ApiRules/ApiRulesList/components.js

```diff
@@ -19,6 +19,15 @@ function navigateToService(apiRule) {
     .navigate(`services/details/${apiRule.spec.service.name}`);
 }
 
+function getGatewayHost(gateway) {
+  const properServer = gateway.spec.servers.filter(
+    server => server.port.protocol === 'HTTPS',
+  );
+  if (!properServer?.length || !properServer[0].hosts?.length) return null;
+  const properHost = properServer[0].hosts[0].replace('*.', '');
+  return properHost;
```

Comment:
you can move return one line above
`return properServer[0].hosts[0].replace('*.', '')`